src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkActions.tsx

```diff
@@ -162,6 +162,7 @@ export const ArtworkActionsFragmentContainer = createFragmentContainer(
       id
       image {
         id
+        url(version: "larger")
```

Comment:
Are you sure we should use larger here? I guess we might need that version for desktop, but definitely not for mobile... hmm...

Reply:
Would the alternative be also requesting a smaller version in the GraphQL query and then specifying which gets passed to Force based on device width?  Agreed that 'larger' is probably not necessary on mobile

Reply:
It is the one that we are currently using for 'images' on artwork page. So it's helpful that is is the same url that would be a background on the image on the page.

Don't think we are requesting a different one based on the screen size. It just gets resized.

Reply:
gotcha, that makes sense to me then. @zephraph good with you if I merge?

Reply:
I think we'll need to just revisit this when we're looking at image optimizations. From my view, the graphql api should'be be responsible for providing the image rendition. That should really be a single URL that can have a specified sized based on the client needs w/ dynamic resizing done via a CDN or service.